dev.html

```diff
@@ -7,8 +7,9 @@
       - https://opendata.datacommons.io (Prod) and https://static.planx-pla.net (QA) to fetch COVID-19 Dashboard data
     frame-src:
       - https://auspice.planx-pla.net and https://auspice.pandemicresponsecommons.org for COVID-19 Auspice app in iframe
+      - https://atlas-qa-mickey.planx-pla.net for Atlas iframe in VHDC qa-mickey
     -->
-    <meta http-equiv="Content-Security-Policy" content="default-src 'self' https://login.bionimbus.org https://wayf.incommonfederation.org; child-src blob:; connect-src 'self' blob: localhost https://localhost:9443 wss://localhost:9443 https://*.s3.amazonaws.com https://*.mapbox.com https://opendata.datacommons.io https://static.planx-pla.net https://*.logs.datadoghq.com https://clinicaltrials.gov; img-src 'self' https://opendata.datacommons.io https://static.planx-pla.net data: https://*.s3.amazonaws.com https://www.google-analytics.com; script-src 'self' 'unsafe-eval' https://www.google-analytics.com localhost https://localhost:9443; worker-src 'self' blob:; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com localhost https://localhost:9443; object-src 'none'; font-src 'self' data: https://fonts.googleapis.com https://fonts.gstatic.com; frame-src 'self' https://auspice.planx-pla.net https://auspice.pandemicresponsecommons.org;">
```

Comment:
why removing https://clinicaltrials.gov from here?

Reply:
I think that was unintentional, @kmhernan ?

Reply:
I'm assuming @alilahrime accepted our branches version over master in the conflict. We had to add the one for atlas iframe. But yeah we shoudl add the clinical trials one back in.

Reply:
Right so just make sure it is included @alilahrime

Reply:
i included it in commit below this "revert dev html.. clinical"